src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -39,6 +50,6 @@ public static long sumAll(List<Integer> numbers) {
     }
 
     public static long sumOverThreeAndDouble(List<Integer> numbers) {
-        return 0;
+        return numbers.stream().filter(number -> number > 3).map(number -> number * 2).reduce(0, (x, y) -> x + y);
```

Comment:
줄바꿈을 통해 가독성을 높여볼 수 있을 것 같아요. 
```suggestion
        return numbers.stream()
            .filter(number -> number > 3)
            .map(number -> number * 2)
            .reduce(0, (x, y) -> x + y);
```